### DISCUSSION TITLE
Merge Google zip files

### DISCUSSION THREAD

**Discussion by Jcode2020** [aonez/Keka#1334] [Q&A] [answered]
Hello everyone,

I would like to download some larger folders from Google Drive. Google first compresses the folder and splits it up, so that I get zip files according to the pattern

My folder-001.zip

Myfolder-002.zip

Myfolder-003.zip

Etc.

Clicking on the first file does not cause the others to be automatically taken into account and if I want to unpack everything at once with the archiving program, I get folders according to the same scheme

Myfolder-001

Myfolder-002

MyFolder-003

I then have to laboriously copy them together by hand and select "merge" when inserting them.

Can I use Keka in a way so that it unpacks and merges all at once?

Thank you!

**Comment by aonez** [comment, marked as answer]
Same as https://github.com/aonez/Keka/discussions/1117. Those are not really split files, rather normal files. I don't really know why Google does that.

The merge process is currently the only option. Even if I implemented something into Keka there should be some type of merge there.

Since I don't even use that service, could you let me know the size of those ZIP files? Maybe I'll do some tests, anyway.
